src/langsmith/agent-builder.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Agent Builder
+sidebarTitle: Overview
+mode: wide
+---
+
+<Callout icon="wand-magic-sparkles" color="#2563EB" iconType="regular">
+Agent Builder is in private preview. Sign up for the waitlist [today](https://www.langchain.com/langsmith-agent-builder-waitlist).
+</Callout>
+
+Agent Builder lets you turn natural-language ideas into production agents. It's powered by [deep-agents](https://github.com/langchain-ai/deepagents), and is not workflow based.
```

Comment:
```suggestion
Agent Builder lets you turn natural-language ideas into production agents. It's powered by [`deepagents`](https://github.com/langchain-ai/deepagents), and is not workflow based.
```

Reply:
This might be my naivety but will the expected users of Agent Builder understand immediately what "not workflow based" means and why it's a positive in this context?

Could be cool to add a tooltip over workflow <tooltip> that just lightly explains that "Predetermined code paths and are designed to operate in a certain order." (from our docs elsewhere

Or, a clarify sentence, something along the lines of:

> Unlike traditional no-code tools that use step-by-step workflows, Agent Builder lets agents decide how to complete tasks. You describe what you want the agent to do in natural language, and the system figures out the steps needed to accomplish it.

I don't know if that is a fair description?